cpp/pagerank_module/pagerank_test.cpp

```diff
@@ -0,0 +1,139 @@
+  auto graph = pagerank_alg::PageRankGraph(1, 0, {});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{1.00};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, TwoNodesOneEdge) {
+  auto graph = pagerank_alg::PageRankGraph(2, 1, {{0, 1}});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.350877362, 0.649122638};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, ZeroNodesZeroEdges) {
+  auto graph = pagerank_alg::PageRankGraph(0, 0, {});
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, OneNodeOneSelfLoop) {
+  auto graph = pagerank_alg::PageRankGraph(1, 1, {{0, 0}});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{1.00};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, TwoNodesTwoMultipleEdges) {
+  auto graph = pagerank_alg::PageRankGraph(2, 2, {{0, 1}, {0, 1}});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.350877362, 0.649122638};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, TwoNodesOneSelfLoop) {
+  auto graph = pagerank_alg::PageRankGraph(2, 1, {{1, 1}});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.130435201, 0.869564799};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, RandomGraph5Nodes10Edges) {
+  auto graph = pagerank_alg::PageRankGraph(
+      5, 10, {{0, 2}, {0, 0}, {2, 3}, {3, 1}, {1, 3}, {1, 0}, {1, 2}, {3, 0}, {0, 1}, {3, 2}});
+
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.240963851, 0.187763717, 0.240963851, 0.294163985, 0.036144598};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, RandomGraph10Nodes10Edges) {
+  auto graph = pagerank_alg::PageRankGraph(
+      10, 10, {{9, 5}, {4, 4}, {3, 8}, {0, 5}, {5, 0}, {3, 0}, {7, 9}, {3, 9}, {0, 4}, {0, 4}});
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.114178360, 0.023587998, 0.023587998, 0.023587998, 0.588577186,
+                                      0.098712132, 0.023587998, 0.023587998, 0.030271265, 0.050321066};
+  ASSERT_TRUE(TestPageRank(results, expected));
+}
+
+TEST(Pagerank, RandomGraph10Nodes9Edges) {
+  auto graph =
+      pagerank_alg::PageRankGraph(10, 9, {{8, 8}, {2, 2}, {0, 8}, {7, 8}, {1, 6}, {0, 0}, {1, 1}, {6, 3}, {9, 5}});
+  auto results = pagerank_alg::ParallelIterativePageRank(graph);
+  auto expected = std::vector<double>{0.047683471, 0.047683471, 0.182781325, 0.067949168, 0.027417774,
+                                      0.050723042, 0.047683471, 0.027417774, 0.473242731, 0.027417774};
```

Comment:
Change everywhere in this file to this:
```suggestion
  std::vector<double> expected{0.047683471, 0.047683471, 0.182781325, 0.067949168, 0.027417774,
                                      0.050723042, 0.047683471, 0.027417774, 0.473242731, 0.027417774};
```